nixos/tests/nik4.nix

```diff
@@ -0,0 +1,69 @@
+# This test downloads some example OpenStreetMap data, imports it into a database and then renders a portion of it. The resulting PNG image will be in the output.
```

Comment:
AFAIK `nixos/tests` is for [features of nixos (so modules)](https://nixos.org/manual/nixos/unstable/#sec-nixos-tests). Your package is not nixos specific, so tests should be a [package test](https://nixos.org/manual/nixpkgs/stable/#var-passthru-tests-packages).

Reply:
But big kudos for writing some tests!

Reply:
Actually, as you need a vm and postgresql, it seems you need a nixos tests. I was wrong assuming it was only for nixos specific thing. It's for tests that needs nixos. Please disregard my comment above!